pkg/operation/botanist/component/extensions/worker/worker.go

```diff
@@ -173,6 +173,7 @@ func (w *worker) deploy(ctx context.Context, operation string) (extensionsv1alph
 			labels = map[string]string{}
 		}
 		labels["node.kubernetes.io/role"] = "node"
+		labels["kubernetes.io/arch"] = *workerPool.Machine.Architecture
```

Comment:
The kubelet also sets this field. Can there be any issues (different values set by us compared to kubelet)?

Reply:
I tested it locally on AWS shoot with a worker pool of both architectures it was working fine. I didn't check the kubelet code that if it overwrites the label or merges it but in any case, there should not be any issue cause the label we are adding is in line with Kubernetes, so even though if it overwrites all the nodes will have `kubernetes.io/arch` label. And if in the future `Kubernetes` changes something we can adapt it on the Gardener side to align with the Kubernetes label.

Reply:
OK, thanks for checking. Then let's continue.

/approve